syft/file/digest_cataloger.go

```diff
@@ -92,15 +92,11 @@ func DigestsFromFile(closer io.ReadCloser, hashes []crypto.Hash) ([]Digest, erro
 		writers[idx] = hashers[idx]
 	}
 
-	size, err := io.Copy(io.MultiWriter(writers...), closer)
+	_, err := io.Copy(io.MultiWriter(writers...), closer)
```

Comment:
We should discuss this change -- since we would be producing invalid SPDX documents even given `digest` enabled and `SHA1` being produced

Reply:
@wagoodman I don't have a ton of context on why 0 size was not getting a digest previously. Let me know if this change is incorrect and breaks another assumption we were making.

Note: we do need to produce digests for files size 0 for a valid SPDX document

Reply:
you're change here is the correct one 👍